.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -15,27 +14,28 @@
+1.  - [ ] Create a OneNote page in [NET Core Acquisition and Deployment XXX Can we link to this?](https://microsoft.sharepoint.com/placeholderlink) > source-build > Servicing > Retrospective > "May 2022" (for example) to take ongoing notes on problems with the process, workarounds, and fixes.
       - This is useful to make sure context is available to review later. It may end up blank if the release goes very smoothly.
       - There are other notes on servicing in this OneNote. It may be useful to review if something goes wrong to see if it's been fixed before.
       - [Non-Internal] File GitHub issues as you encounter problems, and link to them from the notes. Provide info in the issue rather than in the notes.
-1.  - [ ] [Internal] Ensure internal/release/X.Y branch is up to date with mirrored release/X.Y branch.
+1.  - [ ] [Internal] Work in the [dotnet-source-build](https://dnceng.visualstudio.com/internal/_git/dotnet-source-build?path=%2F&version=GBmain&_a=contents) repo.
+    - [ ] [Non-Internal] Work in the [GitHub source-build](https://github.com/dotnet/source-build) repo.
+1.  - [ ] [Internal] Ensure [internal/release/3.1](https://dnceng.visualstudio.com/internal/_git/dotnet-source-build?path=/&version=GBinternal/release/3.1&_a=history) branch is up to date with mirrored [release/3.1](https://github.com/dotnet/source-build/commits/release/3.1) branch.
 1.  - [ ] Update to new version.
-      - [2.1] [/Documentation/servicing/update-2.1.md](https://github.com/dotnet/source-build/tree/release/3.1/Documentation/servicing/update-2.1.md)
       - [3.1] [/Documentation/servicing/update-3.1.md](https://github.com/dotnet/source-build/tree/release/3.1/Documentation/servicing/update-3.1.md)
       - Start a new working branch and PR for WIP so others can easily look if needed and CI starts running.
-        - [Internal] For internal builds, use <https://dev.azure.com/dnceng/internal/_git/dotnet-source-build>, branch name `internal/release/<SDK-version>-<month name><YYYY>`.
+        - [Internal] For internal builds, use the branch name `dev/<youralias>/<SDK-version>-<month name><YYYY>`.  This is necessary because some branch protection kicks in on branches with "release" in the name.
 1.  - [ ] Version update cleanup and confirmations
-      - [2.1]
-        - [ ] Verify that `source-build/ProdConFeed.txt` contains the latest feed required for the release.
-          - Build number should match ProdCon build number.
-          - Ensure auth details are **not** included in this file.
-        - [ ] Verify/update the ASP.NET versions (`MicrosoftAspNetCoreAllPackageVersion` and `MicrosoftAspNetCoreAppPackageVersion`) in dependencies.props.
-        - [ ] Verify/update the `OfficialBuildId`s in updated repos (usually coreclr, corefx, core-setup).
-        - [ ] Update the `DotnetCLIVersion.txt` SDK version to N-1: the latest released "2.1.XYY" SDK version, where "2.1.X" matches the SDK we're building.
       - [3.1]
```

Comment:
Since this is in the 3.1 branch, I think there 3.1 prefixes should be removed.  This one doesn't add value.